src/masoniteorm/collection/Collection.py

```diff
@@ -291,6 +292,9 @@ def reverse(self):
 
     def serialize(self):
         def _serialize(item):
+            if self.__appends__:
+                item.set_appends(self.__appends__)
```

Comment:
I added this if check here because I'm thinking this will break when you go to serialize something that is not a collection of models. If you have a collections lists the list doesn't have set appends